src/components/Images/__snapshots__/Images.spec.js.snap

```diff
@@ -26,3 +26,80 @@ exports[`CloudinaryImage default rendering CloudinaryImage 1`] = `
   />
 </picture>
 `;
+
+exports[`PreloadTags default rendering PreloadTags 1`] = `
```

Comment:
Here's the meat and potatoes, I think this snapshot really shows what this component is about :)

The tags are ordered in smallest to largest screen, smallest to largest dpr
mobile 1.0, mobile 2.0, mobile 3.0, tablet 1.0, ...., desktop 3.0